ParisBNF/abb/BNFabb105.xml

```diff
@@ -0,0 +1,979 @@
+                             <note>Written probably in the main <ref target="#h1">Hand 1</ref>.</note>
+                          </item>
+
+
+
+                          <item xml:id="e1">
+                            <locus target="#3ra"></locus>
+                             <desc type="Unclear"/>
+                             <q xml:lang="gez"></q>
+                             <note>An unclear note.</note>
+                          </item>
+
+                          <item xml:id="e2">
+                            <locus target="#50v"></locus>
+                             <desc type="Unclear"/>
+                         <note>An erased note, mostly unreadable, including a
+                            string of personal names.</note>
+                          </item>
+
+                          <item xml:id="e3">
+                            <locus target="#4r"></locus>
+                             <desc type="StampExlibris"/>
+                         <note>Stamp of the Library.</note>
+                          </item>
+
+                          <item xml:id="e4">
+                            <locus target="#51r"></locus>
+                             <desc type="Unclear"/>
+                             <q xml:lang="gez"></q>
+                             <note>An erased note.</note>
+                          </item>
+
+                          <item xml:id="e5">
+                            <locus target="#194r #194v"></locus>
+                             <desc type="StampExlibris"/>
+                         <note>Stamps, poorly readable.</note>
+                          </item>
+
+                          <item xml:id="e6">
+                            <locus from="195r" to="196v"></locus>
+                             <desc type="Unclear"/>
+                             <q xml:lang="gez"></q>
+                             <note>Multiple exercises in drawing ornamental bands,
+                               scribbles and/or writing exercises.</note>
+                          </item>
+                       </list>
+                      </additions>
+
+                        <bindingDesc>
+                           <binding contemporary="Ethiopian" cert="low">
+                              <decoNote xml:id="b1">Two wooden boards</decoNote>
+                              <decoNote xml:id="b2">Leather quarter cover</decoNote>
+                              <decoNote xml:id="b3" type="SewingStations">4</decoNote>
+                               <decoNote xml:id="b4" type="bindingMaterial">
+                                 <material key="wood"/>
+                                 <material key="leather"/>
+                                    </decoNote>
+                           </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                     <origDate when="1475" cert="medium"/>The
```

Comment:
1475 which calendar? should probably be `notBefore="1482" notAfter="1483"`

Reply:
1482 / 83